presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestRoles.java

```diff
@@ -563,6 +564,119 @@ public void testShowRoleGrants()
                         row("role1"));
     }
 
+    @Test(groups = {ROLES, AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
```

Comment:
https://github.com/prestosql/presto/pull/90/commits/aec0671f2956200c6e562435a601edaad53ce3e3 also renamed `testAccessControl` to `testCreateDropRoleAccessControl` . Although seems unrelated.

Reply:
Yeah. Let's keep the name.